src/core/ConfigManager.cpp

```diff
@@ -40,7 +40,9 @@
 
 // Vector with all the upgrade methods
 const std::vector<ConfigManager::UpgradeMethod> ConfigManager::UPGRADE_METHODS = {
-	&ConfigManager::upgrade_1_1_90    ,    &ConfigManager::upgrade_1_1_91
```

Comment:
I think these should stay on one line since that's what we do in the project file upgrader.

Reply:
To be clear, not all on one line, but 2 per line.